doc/userdoc/guides/nest2_to_nest3/refguide_nest2_nest3.rst

```diff
@@ -304,11 +304,14 @@ divergent *and* num_connections        fixed_outdegree
 Functions related to simulation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+-------------------------+--------------------------------------------+
-| NEST 2.x                | NEST 3.0                                   |
-+=========================+============================================+
-| nest.ResetNetwork()     | Use nest.ResetKernel() instead             |
-+-------------------------+--------------------------------------------+
++-------------------------------+----------------------------------------------+
+| NEST 2.x                      | NEST 3.0                                     |
++===============================+==============================================+
+| nest.ResetNetwork()           | Use nest.ResetKernel() instead               |
++-------------------------------+----------------------------------------------+
+| nest.GetKernelStatus('time'), | nest.GetKernelStatus('biological_time'),     |
```

Comment:
We try to use  :green:\`change\` for the changed code in this ref-guide. Could you update so it is {Set,Get}KernelStatus(':green:\`biological_time\`').
Also, I do not know if this will render as two lines, but maybe that is okay?